### PR TITLE
Project movement model

### PR DESCRIPTION
**Pull request by swellander**
## Description
Creates a new model `ProjectMovement` to replace the `ProjectMetricSummary` model. The new model bundles together the 7/30/90 day % movement summaries for every active metric, as well as includes some info about the project (everything needed to populate the [Leaderboard table](https://www.figma.com/design/SuY5W6zRajPNOrmeCSTvAq/Onchain-Impact-Dashboard?node-id=337-2366&t=ionD6ZYvc3TZyMJT-0))

### CODE REVIEW

File: packages/nextjs/pages/api/projectMetricSummary/index.ts
Comment (swellander) [inline packages/nextjs/pages/api/projectMetricSummary/index.ts:1]: I'm replacing this with a `/api/projectMovement/index.ts` route as part of the [leaderboard](https://github.com/BuidlGuidl/onchain-impact-dashboard/issues/47) ticket

File: packages/nextjs/services/mongodb/models/projectMovement.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  name: { type: String, required: true },
+  category: { type: String, required: true },
+  profileAvatarUrl: { type: String, required: true },
+  movementByMetric: { type: Map, of: MovementByMetricSchema, required: true },
+});
+
+type IProjectMovementModel = Model<IProjectMovement, object>;
+
+export const TempProjectMovement =
+  (mongoose.models.ProjectMovement as IProjectMovementModel) ||
```
Comment (escottalexander) [inline packages/nextjs/services/mongodb/models/projectMovement.ts]: This should be 
```
(mongoose.models.temp_ProjectMovement as IProjectMovementModel) || ...
                 ^^^^^
```

**Review by escottalexander — CHANGES_REQUESTED** [review]
One small comment.

File: packages/nextjs/services/mongodb/models/projectMovement.ts (hunk 1 above)
Comment (swellander) [reply packages/nextjs/services/mongodb/models/projectMovement.ts]: good catch

**Review by escottalexander — APPROVED** [review]

**Pull request merged**